html/js/vr-party-presenter.js

```diff
@@ -44,15 +45,46 @@ function initialize() {
         $('#url').attr('href', url);
         $('#qrcode').qrcode(url);
     
-        _socket.emit('create-session', { id: _sessionId });
-    
-        Autodesk.Viewing.Initializer(getViewingOptions(), function() {
-            launchUrn(_default_models['robot arm']);
-            readCookiesForCustomModel();
-            initializeSelectFilesDialog();
-        });
+        // If the provided session exists...
+        $.get(
+            window.location.origin + '/api/sessionExists/' + _sessionId,
+            function(req1, res1) {
+                if (res1 === "success") {
+                    if (req1) {
+                        // .. then get its data (right now its URN) and load it
+                        $.get(
+                            window.location.origin + '/api/getSession/' + _sessionId,
```

Comment:
I wonder if it makes sense to combine these API endpoints in the server logic:
'/api/sessionExists/'
'/api/getSession/'

If a session does not exist, then it would be created (same logic as here, but simplifies the JS as we only need one round trip)